pyqtgraph/metaarray/MetaArray.py

```diff
@@ -631,7 +631,7 @@ def prettyInfo(self):
         s = ''
         titles = []
         maxl = 0
-        for i in range(len(self._info)-1):
+        for i in range(len(self._info)):
```

Comment:
The last index in `_info` is supposed to be an extra dict containing general meta-info for the entire array. So maybe it should look like:

    for i in range(min(self.ndim, len(self._info))):